it

```diff
@@ -170,11 +157,9 @@ tirare
 topa
 troia
 trombare
-uccello
```

Comment:
We should keep these since we dont want to use them in an autosuggest context. Can you add more info about why you want to remove them?

https://context.reverso.net/traduction/italien-anglais/uccello

<img width="778" alt="Capture d’écran, le 2020-01-01 à 15 13 36" src="https://user-images.githubusercontent.com/196199/71645653-5081cb00-2ca9-11ea-9534-64af97a4db69.png">

Reply:
Because `uccello` is the word for `bird`, it is used more in the first context and this is a list of badwords.
If we need to do a list of slang words that can have different meaning I think that is another task.

Like also `lumaca` that is `snail` and we use that to say that someone is slow.

Reply:
true, google images shows birds:

<img width="1607" alt="Screen Shot 2020-07-13 at 8 06 04 AM" src="https://user-images.githubusercontent.com/196199/87302629-c203c700-c4df-11ea-93bb-58f550e27bd9.png">